.changeset/good-tools-fry.md

```diff
@@ -0,0 +1,17 @@
+---
+"@comet/brevo-api": major
+---
+
+Remove deprecated `sendinblue` package and replace it with `brevo` (https://github.com/getbrevo/brevo-node)
```

Comment:
I think in this case, we really don't need a changeset, since nothing changed for the application.